### PR TITLE
Check PayPal Standard is enabled #62

### LINKED ISSUE #62: PayPal Standard is not active but the option in Woo Viet is active
Error: 

> Notice: Undefined index: description in /wp-content/plugins/woo-viet/inc/class-wooviet-vnd-paypal-standard.php on line 107

From https://wordpress.org/support/topic/onepay-khong-hoat-dong/

### Quick look 

This may happen in the case: 

* “Support VND for the PayPal Standard gateway” is active https://imgur.com/a/QoPkSif
* PayPal Standard is not active in WooCommerce -> Settings

### PR DESCRIPTION
**Pull request by Longkt**
Close #62

### CODE REVIEW

**Comment by htdat** [comment]
Close #62

**Comment by htdat** [comment]
Have you tried testing already? Give me some test cases.

Does this patch not affect this part of the code? 

https://github.com/woocommerce/woocommerce/blob/master/includes/gateways/paypal/class-wc-gateway-paypal.php#L76-L78

File: woo-viet.php [hunk 1]
```diff
@@ -154,6 +154,7 @@ public function main() {
 		}
 
 		$settings = self::get_settings();
+		$woocommerce_paypal_settings = get_option( 'woocommerce_paypal_settings' );
```
Comment (htdat) [inline woo-viet.php]: This code should move prior to L205, which helps increase the clarity of code.

**Comment by Longkt** [comment]
The patch does not affect the code
https://github.com/woocommerce/woocommerce/blob/master/includes/gateways/paypal/class-wc-gateway-paypal.php#L76-L78

Because of this code only check the option of the PayPal gateway.

**Comment by htdat** [comment]
> Have you tried testing already? Give me some test cases.

**Comment by htdat** [comment]
Closing as mentioned here https://github.com/htdat/woo-viet/issues/62#issuecomment-544564245

**Pull request closed without merging**